actors/runtime/src/main/java/cloud/orbit/actors/runtime/Hosting.java

```diff
@@ -394,7 +400,6 @@ public void actorDeactivated(RemoteReference remoteReference)
 
     private void setCachedAddress(final RemoteReference<?> actorReference, final NodeAddress nodeAddress)
```

Comment:
I think this method can be folded in now, there are three places where I think it's more readable that we just invoke put directly on the cache.